backend/src/main/java/zipgo/review/application/ReviewQueryService.java

```diff
@@ -113,112 +112,53 @@ private List<Metadata> findAllBreedSizes() {
-        return rating;
-    }
+        PetFood petFood = petFoodRepository.getById(petFoodId);
+        Reviews reviews = petFood.getReviews();
 
-    private int getZeroOrCalculateRankPercent(int rank, Map<Integer, List<Review>> groupByRating, int reviewSize) {
-        if (reviewSize == 0) {
-            return 0;
-        }
-        return groupByRating.getOrDefault(rank, new ArrayList<>()).size() * PERCENTAGE / reviewSize;
-    }
-
-    private List<SummaryElement> summarizeTastePreference(List<Review> reviews, int reviewSize) {
-        List<SummaryElement> tastePreference = new ArrayList<>();
-        Map<TastePreference, List<Review>> groupByTastePreference = reviews.stream()
-                .collect(Collectors.groupingBy(Review::getTastePreference));
-
-        TastePreference[] values = TastePreference.values();
-        for (TastePreference value : values) {
-            tastePreference.add(new SummaryElement(value.getDescription(),
-                    getZeroOrTastePreferencePercentage(reviewSize, groupByTastePreference, value)));
-        }
-        return tastePreference;
-    }
+        RatingSummaryResponse ratingsSummary = getRatingsSummary(reviews);
+        List<TastePreferenceResponse> tastesSummary = getTastesSummary(reviews);
+        List<StoolConditionResponse> stoolConditionsSummary = getStoolsSummary(reviews);
+        List<AdverseReactionResponse> adverseReactionsSummary = getAdverseReactions(reviews);
 
-    private int getZeroOrTastePreferencePercentage(int reviewSize,
-                                                   Map<TastePreference, List<Review>> groupByTastePreference,
-                                                   TastePreference value) {
-        if (reviewSize == 0) {
-            return 0;
-        }
-        return groupByTastePreference.getOrDefault(value, new ArrayList<>()).size() * PERCENTAGE / reviewSize;
+        return GetReviewsSummaryResponse.of(ratingsSummary, tastesSummary, stoolConditionsSummary, adverseReactionsSummary);
     }
 
-    private List<SummaryElement> summarizeStoolCondition(List<Review> reviews, int reviewSize) {
-        List<SummaryElement> stoolCondition = new ArrayList<>();
-        Map<StoolCondition, List<Review>> groupByStoolCondition = reviews.stream()
-                .collect(Collectors.groupingBy(Review::getStoolCondition));
-
-        StoolCondition[] values = StoolCondition.values();
-        for (StoolCondition value : values) {
-            stoolCondition.add(new SummaryElement(value.getDescription(),
-                    getZeroOrStoolConditionPercentage(reviewSize, groupByStoolCondition, value)));
-        }
-        return stoolCondition;
+    private RatingSummaryResponse getRatingsSummary(Reviews reviews) {
+        double averageRating = reviews.calculateRatingAverage();
+        List<RatingInfoResponse> ratingInfoResponses = IntStream.rangeClosed(1, 5)
```

Comment:
```suggestion
List<RatingInfoResponse> ratingInfoResponses = IntStream.rangeClosed(MIN_RATING, MAX_RATING)
```
상수로 빼주는 건 어떨까요? 원래 저는 이런거 잘 못챙기는 편인데
이 경우에는 가독성이 많이 높아질것 같아서 제안합니당~

Reply:
제가 도메인을 이해한 상태에서 작성해서 그런지 1과 5가 무엇을 뜻하는지 모르는 사람들의 입장에서는 생각하지 못했네요 ㅠ 꼼꼼하게 봐주셔서 감사합니다~